components/infobox/commons/infobox_item.lua

```diff
@@ -38,7 +38,7 @@ function Item:createInfobox()
 					imageDefault = args.default,
 					imageDark = args.imagedark or args.imagedarkmode,
 					imageDefaultDark = args.defaultdark or args.defaultdarkmode,
-					size = args.size
+					size = args.imagesize
```

Comment:
was it checked that this doesn't cause problems?
(might need a bot run)

Reply:
Item is used by Wild Rift (and one page on Mobile Legends)
Skill is used by SC2

Will check pre-merge if the parameters are in use, if so on how many pages and update as needed by either bot or manually (depending on count).

Reply:
Infobox Item doesn't use `|size=` on any page.

Infobox Skill is used on these two
https://liquipedia.net/starcraft2/index.php?title=Template:Infobox_spell&action=edit
https://liquipedia.net/starcraft2/index.php?title=Template:Infobox_ability&action=edit

Reply:
duped the params there (one with old one with new) so imo you can merge